cmsController.php

```diff
@@ -174,10 +186,11 @@ function UpdateArticle ($postData, $db) {
  * @param $postData used to test if function should run and informs what content should be added to DB
  * @param $db to amend
  */
-function DeleteArticle ($postData, $db) {
+function DeleteArticle (array $postData,PDO $db) {
     if(array_key_exists('artDelete',$postData)) {
         $item = $postData['artDelete'];
-        $query = $db->prepare("UPDATE `articles` SET `deleted`=1 WHERE `title`='$item';");
+        $query = $db->prepare("UPDATE `articles` SET `deleted`=1 WHERE `title`=:item;");
+        $query->bindParam(':item',$item);
```

Comment:
i would just do `$query->bindParam(':item', $postData['artDelete']);`

You're not using the variable later elsewhere so you don't really need it...

Reply:
resolved